source/include/common/cellular_common.h

```diff
@@ -208,6 +208,19 @@ typedef CellularPktStatus_t ( * CellularATCommandDataSendPrefixCallback_t ) ( vo
                                                                               char * pLine,
                                                                               uint32_t * pBytesRead );
 
+/**
+ * @ingroup cellular_common_datatypes_functionpointers
+ * @brief Undefined response callback function.
+ *
+ * @param[in] pContext The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] pLine The input line form cellular modem.
+ *
+ * @return CELLULAR_PKT_STATUS_OK if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+typedef CellularPktStatus_t ( * CellularUndefinedRespCallback_t )( const CellularContext_t * pContext,
```

Comment:
Should the context be void* as in the other callback? Who provides this context?

Reply:
Update the in new commit
* _Cellular_RegisterUndefinedRespCallback accept callback context.
* CellularUndefinedRespCallback_t accept callback context.